### DISCUSSION TITLE
The challenge of "lack of structure and support" in open source mentorship programs

### DISCUSSION THREAD

**Discussion by charliemah** [cncf/mentoring#1102] [General]
Hi everyone,

The Teamnova open source project (https://news.ycombinator.com/item?id=37852899) is currently conducting user experience (UX) research, focusing on team-based open source mentorship programs. Your insights and experiences are invaluable to us, and we welcome your input. We are specifically seeking suggestions, advice, and feedback on addressing the challenge of "lack of structure and support" in open source mentorship programs.

This UX research draws inspiration from a significant finding in a study by the Linux Foundation Research titled "Mentorship in Open Source." The study highlights a common issue in mentorship programs: "Many mentorship programs are informal, with mentors and mentees often left to figure out how to work together best. This can make it difficult for mentees to get the support and guidance they need, leading to frustration and disengagement."

We appreciate any contributions you can make towards solving this challenge and enhancing the mentorship experience in open source projects.

**Comment by charliemah** [comment]
While traditional open source mentorship programs are excellent, they are individual-project-based and often lack a structured approach for effective team collaboration.

**Comment by aliok** [comment]
Hey @charliemah 

I wrote my thoughts here: https://cloud-native.slack.com/archives/CGPK98JNQ/p1701746471352329
